staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/types.go

```diff
@@ -547,8 +555,8 @@ var (
 	UintType = newSimpleType("uint", decls.Uint, types.Uint(0))
 
 	// ListType is equivalent to the CEL 'list' type.
-	ListType = NewListType(AnyType)
+	ListType = NewListType(AnyType, noMaxLength)
 
 	// MapType is equivalent to the CEL 'map' type.
-	MapType = NewMapType(AnyType, AnyType)
+	MapType = NewMapType(AnyType, AnyType, noMaxLength)
```

Comment:
how are these used, and does the fact that they embed no length limit impact any use of these?

Reply:
As far as I know `MapType`, `ListType`, etc. are only used in comparisons (such as [here](https://github.com/DangerOnTheRanger/kubernetes/blob/591cb2ea6e8c9b3b417b542e26d3ee0bd8861cff/staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/value.go#L393)), so passing `noMaxLength` should not affect any of those use cases.